js/src/dropdown.js

```diff
@@ -297,7 +298,8 @@ const Dropdown = (($) => {
           preventOverflow: {
             boundariesElement: this._config.boundary
           }
-        }
+        },
+        positionFixed: $(this._menu).hasClass(ClassName.POSITION_FIXED)
```

Comment:
I'm a bit mix-matched because with that Popper.js will apply `position: fixed` but that's already applied by our CSS class `.position-fixed`, what's the difference between ?

Reply:
This param affects `top` and `left` inline style calculataion. If we simply add .position-fixed to a dropdown menu, it would be rendered somewhere in the top left corner of the window, because popper won't know anything about it. 
So, we don't need any classes or styles, we need to configure the PopperJS instance correctly. I'm not sure, what's the best way to do it. Do you think it might be better to use `data-position-fixed' instead of classes?

Reply:
Yep according to your answer it's better to use a parameter instead of a CSS class, because it'll add style